src/queries.rs

```diff
@@ -740,9 +740,11 @@ impl<'a> VintedWrapper<'a> {
         }
 
         if cookie_not_valid {
-            debug!(
+            log::debug!(
```

Comment:
Explain why the `debug` namespace itself is not valid anymore

Reply:
I just forgot to update it from `log:error!`ngl hahahaha